tensorflow/examples/tutorials/word2vec/word2vec_basic.py

```diff
@@ -106,13 +104,13 @@ def generate_batch(batch_size, num_skips, skip_window):
   data_index += span
   for i in range(batch_size // num_skips):
     target = skip_window  # target label at the center of the buffer
```

Comment:
target is meant to refer to the label (see comment here). This change modifies the terminology used. Can we use consistent terminology and always use target to refer to the label (or delete the variable "target" here altogether).